datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/openmetrics.yaml

```diff
@@ -165,6 +165,13 @@
       - type: array
         items:
           type: string
+- name: tag_by_endpoint
```

Comment:
Should the option be at the `init_config` level instead? https://github.com/DataDog/integrations-core/pull/11956#pullrequestreview-964686771

Reply:
Ah, I see. It's fine leaving it as an instance option as long as it's still easy to set for autodiscovery.